types/onscan.js/onscan.js-tests.ts

```diff
@@ -79,3 +79,22 @@ onScan.attachTo(document, {
         return onScan.decodeKeyEvent(oEvent);
     },
 });
+
+// Scan Event
+window.document.addEventListener('scan', onScanSuccess);
+
+function onScanSuccess(scan: ScanEvent) {
```

Comment:
The docs seem to suggest the appropriate signature looks like:

```
document.addEventListener('scan', function(sScanned, iQty){ ... });
```

(where the first argument is a string, the scanned code - rather than an object containing it and the quantity)

Reply:
Not entirely sure how we represent this in the document event map unfortunately :(

Reply:
``onscan.js`` offers two ways to handle a scan: 
1. Defining a callback method in the option of the the ``attachTo()`` function, seen on [lines 41 to 44](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/695868bb135cc0fc70be4d52561f0cb241b34753/types/onscan.js/onscan.js-tests.ts#L41-L44).
2. Using event listeners, which is the subject of this PR.

The ``function(sScanned, iQty){ ... }`` signature is only for the callback function.

The events uses a different signature. The signature itself doesn't seem to be documented, but it can be infered from [the source code which defines the 'scan' event as](https://github.com/axenox/onscan.js/blob/faf262a528ec1923d4f57600aaa6eb86e0e8ba8a/onscan.js#L300-L309):
```js
oEvent = new CustomEvent(
  'scan',
  {	
	  detail: { 
		  scanCode: sScanCode,
		  qty: iSingleScanQty
	  }
  }
);
oDomElement.dispatchEvent(oEvent);
```

Reply:
Ah fair enough. Might be worth submitting a PR to fix their documentation too then!